client/src/components/EditForm.js

```diff
@@ -39,17 +39,16 @@ const EditForm = (props) => {
         const commentId = props.match.params.id;
         const updatedCommentData = { ...commentState };
         updatedCommentData.rating = parseFloat(updatedCommentData.rating);
-
         try {
-        const response = await fetch(
-            `/api/v1/comments/${commentId}`,
-            {
-            method: "PATCH",
-            headers: {
-                "Content-Type": "application/json",
-            },
-            body: JSON.stringify({ comment: updatedCommentData }),
-            }
+            const response = await fetch(
+                `/api/v1/comments/${commentId}`,
+                {
+                method: "PATCH",
+                headers: {
+                    "Content-Type": "application/json",
+                },
+                body: JSON.stringify({ comment: updatedCommentData }),
```

Comment:
fix indentation. 

- the url should be on line 43 with fetch
- line 45 should also be at the end of line 43 (see previous lessons)